src/App/App.js

```diff
@@ -33,8 +33,7 @@ class App extends Component {
 	}
 	
 	getSingleMovie = (id) => {
-		this.setState({ isMovieDetails: true })
-    fetchSingleMovie(id)
+		fetchSingleMovie(id)
     .then(data => {
 			if(typeof data === 'object') {
```

Comment:
We need to get rid of this "check for data" and handle it in the api call so that here we are just telling the app what to do with the single movie data retrieved.